torch/distributed/nn/api/remote_module.py

```diff
@@ -196,9 +196,11 @@ def __init__(
             method = torch.jit.export(method)
             setattr(self, method_name, types.MethodType(method, self))
 
-    def remote_parameters(self, recurse: bool = True) -> List[rpc.RRef[Parameter]]:
-        r"""Returns a list of RRefs of remote module parameters.
+    def remote_parameters(self, recurse: bool = True) -> List[rpc.RRef]:
+        """
+        Returns a list of RRefs of remote module parameters.
```

Comment:
Since we have to change `RRef[Parameter]` to `RRef` in the signature, probably we can add `Parameter` back in the comment to make up this information loss?

For example:
r"""Returns a list of RRefs of remote module parameters (`List[rpc.RRef[Parameter]]`).